src/common/selectors.js

```diff
@@ -448,6 +448,16 @@ export const createGetTags = collectionSelectors => {
   return _extendCollectionSelector(getTags, 'tag')
 }
 
+export const getLastShutdownTime = create(
+  (_, vm) => vm.id,
+  state => state.objects.byType.message,
```

Comment:
Do not pick in the state directly, use the other selectors: `createGetObjectsOfType('message')`.